docs/superpowers/plans/2026-04-29-native-path-api.md

```
@@ -0,0 +1,1370 @@
+    to: ImagePlug,
+    init_opts: [
+      root_url: "http://localhost:4000",
+      param_parser: ImagePlug.ParamParser.Native
+    ]
+
+  match _ do
+    send_resp(conn, 404, "404 Not Found")
+  end
+end
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)